planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/pull_over_planner/pull_over_planner_base.cpp

```diff
@@ -73,7 +73,7 @@ std::optional<PullOverPath> PullOverPath::create(
   double parking_path_max_curvature{};
   std::tie(full_path_curvatures, full_path_max_curvature) = calculateCurvaturesAndMax(full_path);
   std::tie(parking_path_curvatures, parking_path_max_curvature) =
-    calculateCurvaturesAndMax(full_path);
+    calculateCurvaturesAndMax(parking_path);
 
```

Comment:
I was using `full_path` for calculating the curvature of `parking_path` by mistake